tests/RuleSet/SampleRulesBad.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\RuleSet;
```

Comment:
Can we have this (and `SampleRulesOk`) under the `PhpCsFixer\Tests\Fixtures` namespace as it is where all similar stuff (like `FooTransformer` or `StdinFakeStream` are kept)?

Reply:
Sure.